tbx/static_src/sass/components/_promo.scss

```diff
@@ -0,0 +1,83 @@
+@use "config" as *;
+
+.promo-block {
+    display: grid;
+    grid-template-columns: 1fr;
+
+    @include media-query(large) {
+        grid-template-columns: 3fr 2fr;
+        min-height: 630px;
+    }
+
+    &__content {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        background-color: var(--color--light-background);
+        color: var(--color--light-background-text);
+        padding: $spacer-medium $spacer-small;
+
+        @include media-query(large) {
+            padding: $spacer-medium-plus $spacer-half $spacer-half
+                $spacer-medium-plus;
+        }
+    }
+
+    &__title {
+        color: var(--color--light-background-text);
+        margin-bottom: $spacer-mini;
+
+        .motif-heading__drop-cap {
+            background-color: var(--color--light-background-text);
+        }
+
+        @include media-query(large) {
+            margin-bottom: $spacer-small;
+        }
+    }
+
+    &__description {
+        @include font-size(heading-five);
+        margin-bottom: $spacer-medium;
+
+        @include media-query(large) {
+            margin-bottom: $spacer-small-plus;
+        }
+    }
+
+    &__button {
+        width: fit-content;
+        color: var(--color--light-background-text);
+        background-color: transparent;
+        border-color: var(--color--light-background-text);
+    }
+
+    &__link {
```

Comment:
The link needs to be just applied to the text rather than to a big block around it. Could you also use the `link-styles` mixin to apply the underline style.

Reply:
Changed in 80310cbe80e125ed8f8365a1d6b1b205826c82cb